flask-server/main.py

```diff
@@ -247,7 +247,7 @@ def add_clink():
 
     mapping = {
         'clink_id': clink_entity.id,
-        'user_id': request.json['id']
+        'user_id': decode_auth_token(request.json['token'])
```

Comment:
A user has to have access to the dashboard, to create a new clink. My think is that it would be safe to assume jwt is valid because the dashboard verified it.